ortools/linear_solver/xpress_interface_test.cc

```diff
@@ -1307,7 +1354,6 @@ TEST(XpressInterface, CallbackThrowsException) {
 }  // namespace operations_research
 
 int main(int argc, char** argv) {
-  InitGoogle(argv[0], &argc, &argv, true);
```

Comment:
Why remove this ?

Reply:
It brakes the standard googletest arguments apparently without adding functionalities that are useful for us